penguin/llm/model_config.py

```
@@ -4,17 +4,16 @@
 import os
 import time
 import warnings
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from pathlib import Path
 from typing import Any, Dict, Literal, Optional
```

Ruff (0.15.20)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)